src/less/documentation-swedbankpay.less

```diff
@@ -720,6 +720,183 @@
 			}
 		}
 
```

Comment:
Note of the dev:
I am not 100% happy with all this styling.
I think there is clearly room for simplification, reformatting it to have less "1 case rules".
So far we're a bit in a rush, so here it is.
Plus this is documentation website style only, and not even for the components, so it's (in my opinion) much less important to get it super nice and clean.
But I do want to go through it all again and improve & clean what can be 👍